tests/test_simple_nws.py

```diff
@@ -14,6 +14,16 @@
 ZONE = "test_zone"
 
 
+async def test_nws_forecast_parameters(aiohttp_client, mock_urls):
+    app = setup_app()
+    client = await aiohttp_client(app)
+    with pytest.raises(ValueError, match=".* can only be .*") as exc_info:
+        nws = SimpleNWS(*LATLON, USERID, client, forecast_units="usi")
+    assert exc_info.type is ValueError
```

Comment:
This suggestion is slightly better as we no longer need `exc_info`.
```suggestion
    with pytest.raises(ValueError, match=".* can only be .*"):
        nws = SimpleNWS(*LATLON, USERID, client, forecast_units="usi")
```